src/Page/News/NewsItem_.elm

```diff
@@ -18,6 +18,7 @@ import Pages.PageUrl exposing (PageUrl)
 import Shared
 import Theme.Global exposing (viewBackButton, withMediaSmallDesktopUp, withMediaTabletLandscapeUp, withMediaTabletPortraitUp)
 import Theme.PageTemplate as PageTemplate
+import Theme.TransMarkdown exposing (markdownToHtml)
```

Comment:
Apart from css related stuff, we've been using fully qualified imports so it's more  obvious in the markup where stuff comes from.

It's a style / consistency thing and I am sure we've not been particularly consistent anyway.
```suggestion
import Theme.TransMarkdown as TransMarkdown
```

Reply:
good to know